source/content/guides/global-cdn/05-https.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Pantheon Global CDN
+subtitle: Configure DNS and Provision HTTPS
+description: Learn the specifics of Pantheon's Free and Automated HTTPS, powered by Let's Encrypt.
+categories: [go-live]
+tags: [cdn, dns, https, security]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/global-cdn/https
+anchorid: https
+---
+
+Pantheon's [Global CDN](/guides/global-cdn) provides [free, automated HTTPS](https://pantheon.io/features/managed-https) for every site launched on the platform. Your [HTTPS](/guides/global-cdn/https) certificates are fully managed using [Let's Encrypt](https://letsencrypt.org).
+
+<Partial file="configure-dns.md" />
+
+<Alert type="info" title="Note">
+
+When adding the domain to your environment, you may be presented with the option to **Verify your domain to provision HTTPS**. If you're using a manually managed [custom certificate](/custom-certificates#option-2-manually-managed-custom-certificates), skip this step by clicking **Skip to updating DNS**.
+
+</Alert>
+
+For more detailed instructions pertaining to your specific DNS host, click below:
+
+<Accordion title="DNS Host-Specific Instructions" id="host-specific2" icon="info-sign">
+
+<DNSProviderDocs />
+
+If you are having difficulties issuing a [Let's Encrypt](https://letsencrypt.org) certificate you can run diagnostics at [Let's Debug](https://letsdebug.net/). This tool can identify an array of issues specifically for [Let's Encrypt](https://letsencrypt.org) certificates including problems with DNS, nameservers, networking issues, common website misconfigurations, and CA policy issues.
+
+</Accordion>
+
+<Partial file="enable-https.md" />
+
+## Let's Encrypt Certificates
```

Comment:
consider moving this to the top, where it is first referenced

Reply:
The DNS must be configured before the HTTPS is provisioned and the certificates created.